src/pages/on-chain-governance/shared/getTotalVotes.ts

```diff
@@ -0,0 +1,20 @@
+export function getVotePercentage(
+  percYes: number,
+  percNo: number,
+  percNeutral: number,
+  voteDecision: string
```

Comment:
let's return an object of all voteDecision, and remove `voteDecision` as param

Reply:
updated